Sources/OTel/Metrics/OTelMetricProducer/OTLPMetricsDataModel.swift

```diff
@@ -67,37 +63,31 @@ public struct OTelMetricPoint: Equatable, Sendable {
     public var data: OTelMetricData
 }
 
-@_spi(Metrics)
 public struct OTelSum: Equatable, Sendable {
     public var points: [OTelNumberDataPoint]
     public var aggregationTemporality: OTelAggregationTemporailty
     public var monotonic: Bool
 }
 
-@_spi(Metrics)
 public struct OTelGauge: Equatable, Sendable {
     public var points: [OTelNumberDataPoint]
 }
 
-@_spi(Metrics)
 public struct OTelHistogram: Equatable, Sendable {
     public var aggregationTemporality: OTelAggregationTemporailty
     public var points: [OTelHistogramDataPoint]
 }
 
-@_spi(Metrics)
 public struct OTelAttribute: Hashable, Equatable, Sendable {
     public var key: String
     public var value: String
 }
 
-@_spi(Metrics)
 public enum OTelAggregationTemporailty: Equatable, Sendable {
     case delta
     case cumulative
 }
```

Comment:
Discuss: do we need to wrap these enums in a struct for API reasons?

Reply:
Yeah, I think we should, doesn't hurt to do it.

Reply:
OK, I've replaced the each of the public enum types in the datamodel with a package enum wrapped in a public struct. I also removed one of them because we weren't exposing anything useful in the API (the flags).